neuralforecast/core.py

```diff
@@ -24,7 +24,7 @@
     LocalRobustScaler,
     LocalStandardScaler,
 )
-from utilsforecast.compat import DataFrame, Series, pl_DataFrame, pl_Series
+from utilsforecast.compat import DataFrame, DFType, Series, pl_DataFrame, pl_Series
```

Comment:
This was added in utilsforecast 0.2.3, so it has to be updated in the settings.ini

Reply:
Added in fix PR